pkgs/servers/phylactery/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromSourcehut, go }:
+
+stdenv.mkDerivation rec {
```

Comment:
Use `buildGoModule`, see [the docs](https://nixos.org/manual/nixpkgs/unstable/#sec-language-go).

Reply:
That works, thanks!